### PR TITLE
Add the new slides by @matsumana

### PR DESCRIPTION
**Pull request by trustin**
- Added a new article with slides: 'Spring BootユーザのためのArmeria入門'.
- Updated the `<Article />` component to allow specifying more than one type.
  - Updated some articles to take advantage of this feature.
- Updated the `<Article />` component to allow specifying HTML in a description.

### CODE REVIEW

**Comment by trustin** [comment]
@matsumana Would you mind providing the description of your slides/post like [this](https://github.com/line/armeria/commit/07555257c22dfe488da7d5a115e5ef496fb61571#diff-7281feb9c73263c6b94c542d18a55c1d) ? :bow:

File: site/src/pages/community/articles.mdx [hunk 1]
```diff
@@ -141,6 +141,17 @@ Send a pull request by editing [this page](https://github.com/line/armeria/edit/
...
 ### Slides and videos
 
+<Article
+  title="Spring BootユーザのためのArmeria入門"
+  url="https://matsumana.info/blog/2020/07/30/introduce-to-armeria-for-spring-users/"
+  author="松﨑学"
+  date="20200730"
+  type="slides"
+  description="
+    TBW
```
Comment (matsumana) [inline site/src/pages/community/articles.mdx:151]: Thank you for adding!
Could you update with the following?

```
Spring Bootユーザ向けに、以下の内容について説明されています。
・Armeriaが提供する機能、その中でもSREの観点で特に嬉しい機能
・ArmeriaをSpring BootとIntegrationして使う場合のコード例
・Spring Bootで実装されている既存のサービスへのArmeria導入シナリオ例
```

Comment (trustin) [reply site/src/pages/community/articles.mdx]: Updated :bow:

**Comment by matsumana** [comment]
Thank you!

**Pull request merged**